### DISCUSSION TITLE
Possibility of running multiple servers on a same PC

### DISCUSSION THREAD

**Discussion by AWRDev** [h2oai/wave#2313] [Q&A] [answered]
Is it possible to run multiple servers on a same PC and then host different apps on them?
Like if I have different apps/scripts that require restarting of a server I wouldn't need to think about other apps that work fine.

**Comment by mturoci** [comment, marked as answer]
Sure, just start each server on a [different port](https://wave.h2o.ai/docs/configuration/#how-to-start-a-wave-app-on-a-different-port).

**Reply by AWRDev** [reply]
Seems to work exactly as I wished! Thanks, but there is another question 😄: can I disable server logs, maybe except for welcome info?

**Reply by mturoci** [reply]
Sure, set `H2O_WAVE_NO_LOG=1`. See https://wave.h2o.ai/docs/configuration.
